### PR TITLE
[Spanish] Create strings.xml

### PR DESCRIPTION
**Pull request by nmayorga092**
Addition of the Spanish language

### CODE REVIEW

**Comment by z-huang** [comment]
Is this and #120 the same? Is `es` equal with `es-US`?

**Comment by nmayorga092** [comment]
Yes, it is

**Comment by z-huang** [comment]
> Yes, it is

Then I'll merge the one that passes the check first.

File: app/src/main/res/values-es/strings.xml [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+    <string name="sort_by_artist">Artista</string>
+    <string name="sort_by_create_date">Fecha Agregada</string>
+
+    <!-- Snackbar -->
+    <plurals name="snack_bar_delete_song">
+        <item quantity="one">%d canción ha sido eliminada.</item>
+        <item quantity="other">%d canciónes han sido eliminadas.</item>
+    </plurals>
+    <plurals name="n_selected">
+        <item quantity="other">%d seleccionadas</item>
```
Comment (z-huang) [inline app/src/main/res/values-es/strings.xml:131]: For locale "es" (Spanish) the following quantities should also be defined: one

**Review by z-huang — CHANGES_REQUESTED** [review]
One line needs to be added.

File: app/src/main/res/values-es/strings.xml [hunk 2]
```diff
@@ -0,0 +1,156 @@
...
+    <string name="color_cyan">Cian</string>
+    <string name="color_teal">Verde azulado</string>
+    <string name="color_green">Verde</string>
+    <string name="color_light_green">Verde claro</string>
+    <string name="color_lime">Lima</string>
+    <string name="color_yellow">Amarillo</string>
+    <string name="color_amber">Ambar</string>
+    <string name="color_orange">Naranjo</string>
+    <string name="color_deep_orange">Naranjo oscuro</string>
+    <string name="color_brown">Cafe</string>
```
Comment (z-huang) [inline app/src/main/res/values-es/strings.xml:55]: "Cafe" is a common misspelling; did you mean "Café"?

File: app/src/main/res/values-es/strings.xml [hunk 3]
```diff
@@ -0,0 +1,156 @@
...
+    <string name="menu_edit">Editar</string>
+    <string name="menu_song_play_next">Reproducir siguiente</string>
+    <string name="menu_song_add_to_queue">Añadir a la cola</string>
+    <string name="menu_song_add_to_playlist">Agregar a la lista de reproducción</string>
+    <string name="menu_song_download">Descargar</string>
+    <string name="menu_song_remove_download">Quitar descarga</string>
+    <string name="menu_song_delete">Borrar</string>
+
+    <!-- Dialog -->
+    <string name="dialog_title_edit_song">Editar cancion</string>
```
Comment (z-huang) [inline app/src/main/res/values-es/strings.xml:73]: "cancion" is a common misspelling; did you mean "canción"?

**Review by z-huang — CHANGES_REQUESTED** [review]
Some spelling problem, the IDE says.

**Comment by z-huang** [comment]
As #120 has merged, I'll close this PR. Thanks for you contribution!

**Pull request closed without merging**